.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "eslint-config-fluid",
+    "env": {
+        "node": true
+    }
+}
```

Comment:
This project should be linted with respect to itself - so that these newlines get added

Reply:
I guess it already is - and it's just a matter of adding these dotfiles to the JSON task

Reply:
Exactly, they just weren't checked.  I added a rule to cover dot files in the root and fixed the errors revealed.  I also manually tidied up the files not covered by any checks, i.e. the LICENSE.txt and .gitignore.

Reply:
I realised that .gitignore and other top-level dot files were common enough that I should add an "other" category that's only used with the carriage return check.